bin/docker-deploy.sh

```diff
@@ -57,12 +57,19 @@ return 0
 
 start_deploy(){
  KUBE_OPTS=""
- 
+
  if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
    KUBE_OPTS="$KUBE_OPTS --container=$app_name"
  fi
 
- kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"	
+ kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"
+
+ if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
+     for container in `echo $DEPLOYABLE_CONTAINERS | sed -n 1'p' | tr ',' '\n'`
```

Comment:
yuk. Please see how to use arrays in bash https://stackoverflow.com/questions/10586153/split-string-into-an-array-in-bash

Reply:
👍